test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/ExtendedQueryTagTests.cs

```diff
@@ -204,6 +204,39 @@ public async Task GivenInvalidTagLevelInRequestBody_WhenCallingPostAync_ThenShou
         Assert.Equal("The field '$[0].Level' in request body is invalid: Expected value 'Studys' to be one of the following values: ['Instance', 'Series', 'Study']", response.Content.ReadAsStringAsync().Result);
     }
 
+    [Fact]
+    public async Task GivenInvalidDSIS_WhenStoring_ThenServerShouldReturnOK()
+    {
+        DicomTag dsTag = DicomTag.PatientSize;
+        DicomTag isTag = DicomTag.StageNumber;
+        await CleanupExtendedQueryTag(dsTag);
+        await CleanupExtendedQueryTag(isTag);
+        DicomFile dicomFile = Samples.CreateRandomDicomFile();
+        var dataSet = dicomFile.Dataset.NotValidated();
+        dataSet.Add(dsTag, "InvalidDSValue");
+        dataSet.Add(isTag, "InvalidISValue");
+        using DicomWebResponse<DicomDataset> response = await _instanceManager.StoreAsync(dicomFile);
+        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+    }
+
+    [Fact]
+    public async Task GivenInvalidISAndIndexed_WhenStoring_ThenServerShouldReturnConflict()
+    {
+        DicomTag tag = DicomTag.StageNumber;
+
+        await CleanupExtendedQueryTag(tag);
```

Comment:
Will the usage of `StageNumber` in both tests cause race conditions?

Reply:
I will fix it later.